plugins/parsers/registry.go

```diff
@@ -261,8 +260,14 @@ func NewParser(config *Config) (Parser, error) {
 		parser, err = NewPrometheusParser(config.DefaultTags)
 	case "prometheusremotewrite":
 		parser, err = NewPrometheusRemoteWriteParser(config.DefaultTags)
-	case "xml":
-		parser, err = NewXMLParser(config.MetricName, config.DefaultTags, config.XMLConfig)
+	case "xml", "json_xpath", "protobuf":
```

Comment:
What do you think about calling it "protobuf_xpath" to make it similar to "json_xpath"? It also might be wise to reserve "protobuf" for a more native (non-xpath) processor.

Reply:
Changed. This makes all option naming more consistent. I also added the possibility to configure the actual queries using the `xpath` name as this configuration is common between formats.